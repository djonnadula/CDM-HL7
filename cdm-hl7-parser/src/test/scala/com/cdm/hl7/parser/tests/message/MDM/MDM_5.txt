MSH|^~\&|MT_COCQA1A|COCQA1A|DBM||201602111436||MDM^T04|MT_COCQA1A_MDM_QA1ADPTO.1.22960.567|D|2.5||KYA|||
PID|1||J000423598|J500217|Jones^Jessica^Campbell|UNKNOWN|20031230|F|CAGE^JESSICA^CAMPBELL JONES^^^^M|U|456 MARVEL STREET^^FRANKLIN^TN^37064|WILLIAMSON|(615)555-6261|(615)555-6261||S|NON|J00021004075|302-30-2015
PV1|1|O|J.2S|EL|||DRFIRST^FIRST^DOCTOR^^^^^^^^^^XX||DRBRI^DOCTOR^BRITTANY^^^^^^^^^^XX|||||test||Y||CLI||08||||||||||||||||HOM|||COCQA1A||DEP|||201601190932|201601191455|||||||DR.ADAM^SAUNDERS^ADAM^^^^M.D.^^^^^^XX
TXA||PDPNCADMN^Admit Note - Brief|201602111436|201602111434||201602111434|201602111434||DRBRI^DOCTOR^BRITTANY||1TSQBE8554^HAMMOCK^BRITTANY|J.PDOC20160211-0002|||J.PDOC|PDPNCADMN|Signed|N||||DRBRI^DOCTOR^BRITTANY^^^^^^^^^^^^201602111434
OBX|1|TX||| 
OBX|2|TX|||MEDICAL CENTER (COCQA1A)
OBX|3|TX|||Admit Note - Brief
OBX|4|TX|||REPORT#:0211-0002  REPORT STATUS: Signed
OBX|5|TX|||DATE:02/11/16 TIME: 1434
OBX|6|TX|||  
OBX|7|TX|||PATIENT: Jones,Jessica Campbell         UNIT #: J000423598
OBX|8|TX|||ACCOUNT#: J00021004075                 ROOM/BED:
OBX|9|TX|||DOB: 12/30/03  AGE: 12      SEX: F      ATTEND: DOCTOR FIRST
OBX|10|TX|||ADM DT:                                 AUTHOR: DOCTOR,BRITTANY
OBX|11|TX|||REP SRV DT: 02/11/16                REP SRV TM: 1434
OBX|12|TX|||* ALL edits or amendments must be made on the electronic/computer document *
OBX|13|TX||| 
OBX|14|TX||| 
OBX|15|TX|||Electronically Signed by DOCTOR,BRITTANY on 02/11/16 at 1436
OBX|16|TX|||  
OBX|17|TX|||RPT #: 0211-0002
OBX|18|TX|||***END OF REPORT***
OBX|19|TX|||  
OBX|20|TX||| 

