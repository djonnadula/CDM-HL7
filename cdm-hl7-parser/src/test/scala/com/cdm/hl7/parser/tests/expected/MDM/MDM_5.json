{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "MT_COCQA1A_MDM_QA1ADPTO.1.22960.567",
    "medical_record_num": "J000423598",
    "medical_record_urn": "J500217",
    "patient_account_num": "J00021004075"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "MT_COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "DBM",
    "receiving_facility": null,
    "date_time_of_message": "201602111436",
    "security": null,
    "message_type": {
      "message_code": "MDM",
      "trigger_event": "T04"
    },
    "message_control_id": "MT_COCQA1A_MDM_QA1ADPTO.1.22960.567",
    "processing_id": "D",
    "version_id": "2.5",
    "sequence_num": null,
    "continuation_pointer": "KYA"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": "J000423598",
    "alternate_patient_id": "J500217",
    "patient_name": {
      "family_name": "Jones",
      "given_name": "Jessica",
      "second_and_further_given_names_or_initials_thereof": "Campbell"
    },
    "mother_maiden_name": "UNKNOWN",
    "date_time_of_birth": "20031230",
    "administrative_sex": "F",
    "patient_alias": {
      "family_name": "CAGE",
      "given_name": "JESSICA",
      "second_and_further_given_names_or_initials_thereof": "CAMPBELL JONES",
      "suffix": null,
      "prefix": null,
      "degree": null,
      "name_type_code": "M"
    },
    "race": "U",
    "patient_address": {
      "street_address": "456 MARVEL STREET",
      "other_designation": null,
      "city": "FRANKLIN",
      "state_or_province": "TN",
      "zip_or_postal_code": "37064"
    },
    "county_code": "WILLIAMSON",
    "phone_num_home": "(615)555-6261",
    "phone_num_business": "(615)555-6261",
    "primary_language": null,
    "marital_status": "S",
    "religion": "NON",
    "patient_account_num": "J00021004075",
    "ssn_num_patient": "302-30-2015"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "O",
    "assigned_patient_location": "J.2S",
    "admission_type": "EL",
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "DRFIRST",
      "family_name": "FIRST",
      "given_name": "DOCTOR",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    },
    "referring_doctor": null,
    "primary_care_doctor": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    },
    "hospital_service": null,
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": "test",
    "mode_of_arrival": null,
    "vip_indicator": "Y",
    "admitting_doctor": null,
    "patient_type": "CLI",
    "visit_num": null,
    "financial_class": "08",
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": "HOM",
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": "COCQA1A",
    "reason_for_visit": null,
    "account_status": "DEP",
    "pending_location": null,
    "prior_temporary_location": null,
    "admit_date_time": "201601190932",
    "discharge_date_time": "201601191455",
    "current_patient_balance": null,
    "total_charges": null,
    "total_adjustments": null,
    "total_payments": null,
    "alternate_visit_id": null,
    "visit_indicator": null,
    "consulting_doctor": {
      "person_id": "DR.ADAM",
      "family_name": "SAUNDERS",
      "given_name": "ADAM",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": "M.D.",
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    }
  },
  "0004.TXA": {
    "set_id": null,
    "document_type": {
      "id": "PDPNCADMN",
      "text": "Admit Note - Brief"
    },
    "document_content_presentation": "201602111436",
    "activity_date_time": "201602111434",
    "primary_activity_provider_code_name": null,
    "origination_date_time": "201602111434",
    "transcription_date_time": "201602111434",
    "edit_date_time": null,
    "originator_code_name": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY"
    },
    "assigned_document_authenticator": null,
    "transcriptionist_code_name": {
      "person_id": "1TSQBE8554",
      "family_name": "HAMMOCK",
      "given_name": "BRITTANY"
    },
    "unique_document_num": "J.PDOC20160211-0002",
    "parent_document_num": null,
    "placer_order_num": null,
    "filler_order_num": "J.PDOC",
    "unique_document_file_name": "PDPNCADMN",
    "document_completion_status": "Signed",
    "document_confidentiality_status": "N",
    "document_availability_status": null,
    "document_storage_status": null,
    "document_change_reason": null,
    "authentication_person_time_stamp": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": null,
      "assigning_facility": null,
      "date_time_action_performed": "201602111434"
    }
  },
  "0005.OBX": {
    "set_id": "1",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": " "
  },
  "0006.OBX": {
    "set_id": "2",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "MEDICAL CENTER (COCQA1A)"
  },
  "0007.OBX": {
    "set_id": "3",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "Admit Note - Brief"
  },
  "0008.OBX": {
    "set_id": "4",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "REPORT#:0211-0002  REPORT STATUS: Signed"
  },
  "0009.OBX": {
    "set_id": "5",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "DATE:02/11/16 TIME: 1434"
  },
  "0010.OBX": {
    "set_id": "6",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "  "
  },
  "0011.OBX": {
    "set_id": "7",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "PATIENT: Jones,Jessica Campbell         UNIT #: J000423598"
  },
  "0012.OBX": {
    "set_id": "8",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "ACCOUNT#: J00021004075                 ROOM/BED:"
  },
  "0013.OBX": {
    "set_id": "9",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "DOB: 12/30/03  AGE: 12      SEX: F      ATTEND: DOCTOR FIRST"
  },
  "0014.OBX": {
    "set_id": "10",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "ADM DT:                                 AUTHOR: DOCTOR,BRITTANY"
  },
  "0015.OBX": {
    "set_id": "11",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "REP SRV DT: 02/11/16                REP SRV TM: 1434"
  },
  "0016.OBX": {
    "set_id": "12",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "* ALL edits or amendments must be made on the electronic/computer document *"
  },
  "0017.OBX": {
    "set_id": "13",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": " "
  },
  "0018.OBX": {
    "set_id": "14",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": " "
  },
  "0019.OBX": {
    "set_id": "15",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "Electronically Signed by DOCTOR,BRITTANY on 02/11/16 at 1436"
  },
  "0020.OBX": {
    "set_id": "16",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "  "
  },
  "0021.OBX": {
    "set_id": "17",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "RPT #: 0211-0002"
  },
  "0022.OBX": {
    "set_id": "18",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "***END OF REPORT***"
  },
  "0023.OBX": {
    "set_id": "19",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": "  "
  },
  "0024.OBX": {
    "set_id": "20",
    "value_type": "TX",
    "obsv_id": null,
    "obsv_sub_id": null,
    "obsv_value": " "
  }
}